src/main/java/mate/academy/model/ShoppingCart.java

```diff
@@ -0,0 +1,78 @@
+package mate.academy.model;
+
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.OneToMany;
+import jakarta.persistence.OneToOne;
+import jakarta.persistence.Table;
+import java.util.List;
+import java.util.Objects;
+
+@Entity
+@Table(name = "shoppingcarts")
+public class ShoppingCart {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+    @OneToMany
+    private List<Ticket> tickets;
+    @OneToOne
```

Comment:
1) `@OneToOne` relation has EAGER fetch type by default, we want to use LAZY
2) Since shoppint cart can't exist without User, they can have shared primary key. We can use `@MapsId` for this
```suggestion
    @MapsId
    @OneToOne(fetch = FetchType.LAZY)
```